src/Conference/ConferenceDetails.test.js

```diff
@@ -1,8 +1,14 @@
 import ConferenceDetails from "./ConferenceDetails";
-import { configure, shallow } from "enzyme";
+import { configure, shallow, mount } from "enzyme";
```

Comment:
Forgot to mention earlier, you're not using shallow anymore, so you can remove the import here ;)